rails_app/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN mkdir -p /app
 WORKDIR /app
 
 COPY Gemfile Gemfile.lock ./
-RUN gem install bundler && bundle install --jobs 4 --retry 5
```

Comment:
Hi! I've reached your change by chance, and I tried it, but it seems to fail with a `libv8` dependency.  Did you have this error with your change?

Reply:
No, its works normal after the change, but i will try later again